app/src/main/java/com/example/unlibrary/library/LibraryBooksFragment.java

```diff
@@ -6,21 +6,35 @@
 import android.view.View;
 import android.view.ViewGroup;
 
+import androidx.databinding.DataBindingUtil;
+import androidx.databinding.ViewDataBinding;
 import androidx.fragment.app.Fragment;
+import androidx.lifecycle.Observer;
 import androidx.lifecycle.ViewModelProvider;
 import androidx.recyclerview.widget.LinearLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
 
 import com.example.unlibrary.R;
+import com.example.unlibrary.models.Book;
+
+import java.util.ArrayList;
+
+/*
+ * LibraryBooksFragment
+ *
+ * October 22, 2020
+ *
+ * Copyright (c) Team 24, Fall2020, CMPUT301, University of Alberta
+ */
 
 /**
  * A fragment representing a list of Items.
  */
 public class LibraryBooksFragment extends Fragment {
 
     private static final String ARG_COLUMN_COUNT = "column-count";
-
     private LibraryViewModel mViewModel;
+    private Observer<ArrayList<Book>> bookListObserver;
```

Comment:
Let's follow best Android code convention best practices. Private variables should be prefixed with m.
```suggestion
    private Observer<ArrayList<Book>> mBookListObserver;
```